content/about/main/index.md

```diff
@@ -7,27 +7,27 @@ number_categories: 3 # set to zero to exclude
 show_intro: true
 intro: |
 
-  I was born in 1977 in [Juan José Paso](https://es.wikipedia.org/wiki/Juan_Jos%C3%A9_Paso_(Buenos_Aires)), a small town in [Argentina](https://es.wikipedia.org/wiki/Argentina). I was six years old when my elementary school bought a computer (one computer for the entire school and the only one in the whole town). The moment I saw it I felt love at first sight.
+  I was born in 1977 in [Juan José Paso](https://es.wikipedia.org/wiki/Juan_Jos%C3%A9_Paso_(Buenos_Aires)), a small town in [Argentina](https://es.wikipedia.org/wiki/Argentina). I was six year old when my elementary school bought a computer (one computer for the entire school and the only one in the whole town). The moment I saw it I felt love at first sight.
```

Comment:
From your RStudio talk I remember there's a cool picture of you at that age or later in the other class?

Reply:
(that you could add here :grin: )